### PR TITLE
Soar 17022 InsightIDR Fix Schema

### PR DESCRIPTION
**Pull request by jerichardson-r7**
### Description

Describe the proposed changes:

  - Update SDK.
  - Bump aiohttp to fix dependency issue.
  - Fix issue where message can be received as an object or a string.

https://rapid7.atlassian.net/browse/SOAR-17022

### Testing

- [x] manual testing of plugin
- [x] unit tests all passing

### CODE REVIEW

File: plugins/rapid7_insightidr/help.md [hunk 1]
```diff
@@ -959,39 +959,41 @@ Example output:
...
-        "id": "bc38a911-65f1-4755-cca3-a330a6336b3a",
-        "links": [
-          {
-            "href": "https://example.com/3e966a63-bf3a-4a3c-8903-979c7e90ce85",
-            "rel": "Self"
-          }
-        ],
-        "name": "Unparsed Data",
-        "rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:logset:bc38a911-65f1-4755-cca3-a330a6336b3a"
+    "log": {
```
Comment (igorski-r7) [inline plugins/rapid7_insightidr/help.md:962]: Just wondering why its so nested? `log` in `log`?

File: plugins/rapid7_insightidr/help.md [hunk 2]
```diff
@@ -1419,46 +1421,48 @@ Example output:
...
-            "links": [
-              {
-                "href": "https://example.com/3e966a63-bf3a-4a3c-8903-979c7e90ce85",
-                "rel": "Self"
-              }
-            ],
-            "name": "Unparsed Data",
-            "rrn": "rrn:logsearch:us:bc38a911-65f1-4755-cca3-a330a6336b3a:logset:bc38a911-65f1-4755-cca3-a330a6336b3a"
+  "logs": {
+    "logs": [
```
Comment (igorski-r7) [inline plugins/rapid7_insightidr/help.md:1425]: Same here

File: plugins/rapid7_insightidr/help.md (hunk 1 above)
Comment (igorski-r7) [reply plugins/rapid7_insightidr/help.md:962]: Are the examples in `plugin.spec.file` look fine?

Comment (jerichardson-r7) [reply plugins/rapid7_insightidr/help.md]: fixed 🙂

File: plugins/rapid7_insightidr/help.md (hunk 2 above)
Comment (jerichardson-r7) [reply plugins/rapid7_insightidr/help.md]: fixed 🙂

Comment (jerichardson-r7) [reply plugins/rapid7_insightidr/help.md]: its very annoying

**Comment by rbowden-r7** [comment]
would it be possible to also bump aiohttp in this release as well ?
https://github.com/rapid7/insightconnect-plugins/pull/2490

**Comment by jerichardson-r7** [comment]
> would it be possible to also bump aiohttp in this release as well, to address this dependabot pr? #2490

Yep no probs! Done 👍

**Review by igorski-r7 — APPROVED** [review]

**Review by ablakley-r7 — APPROVED** [review]

**Review by ekelly-r7 — APPROVED** [review]

**Pull request merged**